Marlin/ubl_motion.cpp

```diff
@@ -477,7 +472,7 @@
             SERIAL_ECHOPAIR("  err=",fabs(z_optimized-z0));
             SERIAL_ECHO("\n");
           }
-        //*/
+        */
```

Comment:
It allows commenting-in the block in question by adding a single `/` to the initial comment tag.